console/v1/types.go

```diff
@@ -7,3 +7,22 @@ type Link struct {
 	// href is the absolute secure URL for the link (must use https)
 	Href string `json:"href"`
 }
+
+//  Represents ability to provide custom pod log links.
+type LogLink struct {
```

Comment:
Wasn't really sure if w want to have it in the `types.go` or directly `types_console_log_links.go`, but since it was similar to the `Link` struct, I've put it here.

Reply:
Or maybe:
- `LogLinkTemplate`
- `PodLogLink`
- `PodLogLinkTemplate`

Wasn't sure regarding the name here..

Reply:
I put `Link` here as it is shared among several of the other resources.  I don't think this one will be shared, so it may be best in `types_console_log_links.go`.

The naming is tricky agree. Since this one is embedded, it doesn't matter quite as much, but it would be good if it was clear.

Reply:
This will eventually be generic for other types, so I would avoid using "pod" in the name.